charts/capi-runtime-extensions/templates/cluster-autoscaler/manifests/helm-addon-installation.yaml

```
@@ -0,0 +1,51 @@
+# Copyright 2024 D2iQ, Inc. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+{{- if .Values.hooks.clusterAutoscaler.helmAddonStrategy.defaultValueTemplateConfigMap.create }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: '{{ .Values.hooks.clusterAutoscaler.helmAddonStrategy.defaultValueTemplateConfigMap.name }}'
+data:
+  values.yaml: |-
+    ---
+    # This is a hack, but because a single cluster-autoscaler deployment can only monitor a single Kubernetes cluster
+    # we expect 'tmpl-clustername-tmpl' and 'tmpl-clusternamespace-tmpl'
+    # to be replaced with the Cluster's name and namespace.
+    fullnameOverride: cluster-autoscaler-tmpl-clustername-tmpl
```

I think we can use variables here to use the Helm addon provider's templating:

```suggestion
    fullnameOverride: "cluster-autoscaler-{{ .Cluster.Name }}"
```

---

Actually  needs to be 
```suggestion
    fullnameOverride: cluster-autoscaler-{{ .Cluster.metadata.name }}
```

---

Done.